eng/build.ps1

```diff
@@ -539,8 +539,13 @@ function EnablePreviewSdks() {
 # Deploy our core VSIX libraries to Visual Studio via the Roslyn VSIX tool.  This is an alternative to
 # deploying at build time.
 function Deploy-VsixViaTool() {
-  $vsixDir = Get-PackageDir "RoslynTools.VSIXExpInstaller"
-  $vsixExe = Join-Path $vsixDir "tools\VsixExpInstaller.exe"
+
+  $vsixExe = Join-Path $ArtifactsDir "bin\Microsoft.VisualStudio.LanguageServices.New.IntegrationTests\$configuration\net472\VSIXExpInstaller\VSIXExpInstaller.exe"
```

Comment:
Now that we're not building on the same machine, the nuget package directory is not available.  Adding the vsix installer to the integration test project is an easy way to ensure that it is available in the test payload.

Reply:
❗ This is an unsupported VSIX installer, and really needs to be removed from usage. The new integration tests already reference the new one, so we definitely should not be using that project to set this bad example. The solution here is to package up the solution packages folder separately from other build artifacts, similar to what I showed in the other example PR I sent you.